packages/scheduler/src/forks/SchedulerDOM.js

```diff
@@ -217,17 +176,11 @@ function workLoop(hasTimeRemaining, initialTime) {
       currentTask.callback = null;
       currentPriorityLevel = currentTask.priorityLevel;
       const didUserCallbackTimeout = currentTask.expirationTime <= currentTime;
-      markTaskRun(currentTask, currentTime);
```

Comment:
We had some unwrapped profiling method calls. Wonder if that was costing us bytes?